index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="styel.css" />
+  </head>
+  <body>
+    <div class="nav-container">
+      <div class="wrapper">
+        <nav>
+          <h2 class="logo">Logo</h2>
+          <img  class="image" src="Enabled.svg" alt="" />
+        </nav>
+      </div>
+    </div>
+    <div class="header-container">
+      <div class="wrapper">
+        <header>
```

Comment:
- [ ] Kindly, instead of using the `<div>` tag as the container of the headline section, It'll be better to use the `<section>` tag. please, solve this issue and then remove the `<header>` tag inside this section.